items:
- uid: System.Resources.MissingManifestResourceException
  commentId: T:System.Resources.MissingManifestResourceException
  id: MissingManifestResourceException
  parent: System.Resources
  children:
  - System.Resources.MissingManifestResourceException.#ctor
  - System.Resources.MissingManifestResourceException.#ctor(System.String)
  - System.Resources.MissingManifestResourceException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  - vb
  name: MissingManifestResourceException
  fullName: System.Resources.MissingManifestResourceException
  type: Class
  source:
    remote:
      path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
      branch: master
      repo: https://github.com/dotnet/corefx.git
    id: MissingManifestResourceException
    path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
    startLine: 10
  assemblies:
  - System.Resources.ResourceManager
  namespace: System.Resources
  summary: "\n<p>The exception that is thrown if the main assembly does not contain the resources for the neutral culture, and an appropriate satellite assembly is missing.</p>\n"
  syntax:
    content: 'public class MissingManifestResourceException : Exception'
    content.vb: >-
      Public Class MissingManifestResourceException
          Inherits Exception
  seealso:
  - type: System.Exception
    commentId: T:System.Exception
  - type: System.Resources.ResourceManager
    commentId: T:System.Resources.ResourceManager
  inheritance:
  - System.Object
  modifiers.csharp:
  - public
  - class
  modifiers.vb:
  - Public
  - Class
- uid: System.Resources.MissingManifestResourceException.#ctor
  commentId: M:System.Resources.MissingManifestResourceException.#ctor
  id: '#ctor'
  parent: System.Resources.MissingManifestResourceException
  langs:
  - csharp
  - vb
  name: MissingManifestResourceException()
  fullName: System.Resources.MissingManifestResourceException.MissingManifestResourceException()
  type: Constructor
  source:
    remote:
      path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
      branch: master
      repo: https://github.com/dotnet/corefx.git
    id: .ctor
    path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
    startLine: 13
  assemblies:
  - System.Resources.ResourceManager
  namespace: System.Resources
  summary: "\n<p>Initializes a new instance of the <xref href=\"System.Resources.MissingManifestResourceException\" data-throw-if-not-resolved=\"false\"></xref> class with default properties.</p>\n"
  remarks: "\n<p>The following table shows the initial property values for an instance of <xref href=\"System.Resources.MissingManifestResourceException\" data-throw-if-not-resolved=\"false\"></xref>.</p>\n<p>Property </p>\n<p>Value </p>\n<table><tbody><tr><td>\n<p>\n<xref href=\"System.Exception.InnerException\" data-throw-if-not-resolved=\"false\"></xref>\n</p>\n</td><td>\n<p>\n<strong>null</strong>. </p>\n</td></tr><tr><td>\n<p>\n<xref href=\"System.Exception.Message\" data-throw-if-not-resolved=\"false\"></xref>\n</p>\n</td><td>\n<p>The localized error message for <xref href=\"System.Resources.MissingManifestResourceException\" data-throw-if-not-resolved=\"false\"></xref>. </p>\n</td></tr></tbody></table>\n"
  syntax:
    content: public MissingManifestResourceException()
    content.vb: Public Sub New
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: System.Resources.MissingManifestResourceException.#ctor(System.String)
  commentId: M:System.Resources.MissingManifestResourceException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Resources.MissingManifestResourceException
  langs:
  - csharp
  - vb
  name: MissingManifestResourceException(String)
  fullName: System.Resources.MissingManifestResourceException.MissingManifestResourceException(System.String)
  type: Constructor
  source:
    remote:
      path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
      branch: master
      repo: https://github.com/dotnet/corefx.git
    id: .ctor
    path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
    startLine: 14
  assemblies:
  - System.Resources.ResourceManager
  namespace: System.Resources
  summary: "\n<p>Initializes a new instance of the <xref href=\"System.Resources.MissingManifestResourceException\" data-throw-if-not-resolved=\"false\"></xref> class with the specified error message.</p>\n"
  remarks: "\n<p>The following table shows the initial property values for an instance of <xref href=\"System.Resources.MissingManifestResourceException\" data-throw-if-not-resolved=\"false\"></xref>.</p>\n<p>Property </p>\n<p>Value </p>\n<table><tbody><tr><td>\n<p>\n<xref href=\"System.Exception.InnerException\" data-throw-if-not-resolved=\"false\"></xref>\n</p>\n</td><td>\n<p>\n<strong>null</strong>. </p>\n</td></tr><tr><td>\n<p>\n<xref href=\"System.Exception.Message\" data-throw-if-not-resolved=\"false\"></xref>\n</p>\n</td><td>\n<p>The <em>message</em> string. </p>\n</td></tr></tbody></table>\n"
  syntax:
    content: public MissingManifestResourceException(string message)
    content.vb: Public Sub New(message As String)
    parameters:
    - id: message
      type: System.String
      description: "\n<p>The error message that explains the reason for the exception. </p>\n"
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
- uid: System.Resources.MissingManifestResourceException.#ctor(System.String,System.Exception)
  commentId: M:System.Resources.MissingManifestResourceException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Resources.MissingManifestResourceException
  langs:
  - csharp
  - vb
  name: MissingManifestResourceException(String, Exception)
  fullName: System.Resources.MissingManifestResourceException.MissingManifestResourceException(System.String, System.Exception)
  type: Constructor
  source:
    remote:
      path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
      branch: master
      repo: https://github.com/dotnet/corefx.git
    id: .ctor
    path: src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.cs
    startLine: 15
  assemblies:
  - System.Resources.ResourceManager
  namespace: System.Resources
  summary: "\n<p>Initializes a new instance of the <xref href=\"System.Resources.MissingManifestResourceException\" data-throw-if-not-resolved=\"false\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.</p>\n"
  remarks: "\n<p>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref href=\"System.Exception.InnerException\" data-throw-if-not-resolved=\"false\"></xref> property. The <xref href=\"System.Exception.InnerException\" data-throw-if-not-resolved=\"false\"></xref> property returns the same value that is passed into the constructor, or <strong>null</strong> if the <xref href=\"System.Exception.InnerException\" data-throw-if-not-resolved=\"false\"></xref> property does not supply the inner exception value to the constructor.</p>\n<p>The following table shows the initial property values for an instance of <xref href=\"System.Resources.MissingManifestResourceException\" data-throw-if-not-resolved=\"false\"></xref>.</p>\n<p>Property </p>\n<p>Value </p>\n<table><tbody><tr><td>\n<p>\n<xref href=\"System.Exception.InnerException\" data-throw-if-not-resolved=\"false\"></xref>\n</p>\n</td><td>\n<p>The inner exception reference. </p>\n</td></tr><tr><td>\n<p>\n<xref href=\"System.Exception.Message\" data-throw-if-not-resolved=\"false\"></xref>\n</p>\n</td><td>\n<p>The error message string. </p>\n</td></tr></tbody></table>\n"
  syntax:
    content: public MissingManifestResourceException(string message, Exception inner)
    content.vb: Public Sub New(message As String, inner As Exception)
    parameters:
    - id: message
      type: System.String
      description: "\n<p>The error message that explains the reason for the exception. </p>\n"
    - id: inner
      type: System.Exception
      description: "\n<p>The exception that is the cause of the current exception. If the <em>inner</em> parameter is not <strong>null</strong>, the current exception is raised in a <strong>catch</strong> block that handles the inner exception. </p>\n"
  seealso:
  - type: System.Exception
    commentId: T:System.Exception
  modifiers.csharp:
  - public
  modifiers.vb:
  - Public
references:
- uid: System.Resources
  isExternal: false
  name: System.Resources
  fullName: System.Resources
- uid: System.Object
  parent: System
  isExternal: false
  name: Object
  fullName: System.Object
- uid: System
  isExternal: false
  name: System
  fullName: System
- uid: System.String
  parent: System
  isExternal: false
  name: String
  fullName: System.String
- uid: System.Exception
  parent: System
  isExternal: false
  name: Exception
  fullName: System.Exception
